django/core/validators.py

```diff
@@ -394,11 +394,33 @@ def compare(self, a, b):
 
 @deconstructible
 class StepValueValidator(BaseValidator):
-    message = _("Ensure this value is a multiple of step size %(limit_value)s.")
+    message = _(
+        "Ensure this value is a multiple of step size %(limit_value)s, starting from "
```

Comment:
Can we have negative numbers? The „starting from“ bit might need to be optional

Reply:
Theoretically we can, but does it change anything? For example _"Ensure this value is a multiple of step size -1.2, starting from 2.2, e.g. 2.2, 1.0, -0.2, and so on."_ sounds good to me.

Reply:
Sorry I mean if the step size it 2 and there is no min_value, would it imply you can’t use -2 because it says „starting from 0, e.g 0, 2, 4“

Reply:
We also have a max_value, I guess that should be used when the step is negative (hadn't thought of negative steps). 
The current offset can be outside the range of `min_value` and `max_value` (e.g. there's no offset and min_value=10, max_value=20).
Maybe the error message is still not designed so well, it's not trivial to give a valid range

Reply:
> Sorry I mean if the step size it 2 and there is no min_value, would it imply you can’t use -2 because it says „starting from 0, e.g 0, 2, 4“

Agreed, we can restore the previous message in this case. I pushed edits.

Reply:
> We also have a max_value, I guess that should be used when the step is negative (hadn't thought of negative steps).

:thinking: As far as I'm aware `max_value` shouldn't have any impact on steps and valid values, it doesn't have in [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/step).

Reply:
> The current offset can be outside the range of `min_value` and `max_value` (e.g. there's no offset and limit_value=2, min_value=10, max_value=20).

I'm not sure if I understand, for built-in fields `min_value` is passed as an `offset`.